VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -187,20 +192,19 @@ public void addAll(List<DiscussionComment> items) {
         notifyItemChanged(lastCommentIndex); // Last item's background is different, so must be refreshed as well
     }
 
-    private static class ViewHolder extends RecyclerView.ViewHolder {
+    private static class ResponseAndCommentViewHolder extends RecyclerView.ViewHolder {
         View discussionCommentRow;
         TextView discussionCommentBody;
-        TextView discussionCommentAuthorTextView;
         TextView discussionCommentCountReportTextView;
-        TextView responseAnswerTextView;
 
-        public ViewHolder(View itemView) {
+        AuthorLayoutViewHolder authorLayoutViewHolder;
```

Comment:
All these fields in the view holder should be declared to be `final`. It would probably be semantically better to also declare them as `public`, since they're supposed to be public. It doesn't really make a difference here though, since it's declared as an inner private class. Same goes for the view holders in the other adapters.